xml_inscriptions/transcribed/IN.Bloom.IUAM.G.63.105.84c.xml

```diff
@@ -25,15 +25,15 @@
                <msContents>
                   <textLang mainLang="grc" otherLangs=""/>
                   <msItem class="#magic">
-                     <p>Chnoubis and Solar Deity</p>
```

Comment:
I wonder if we should add this to the decoration description down below, if you're going to remove it here?